ci-operator/templates/openshift/installer/cluster-launch-installer-e2e-temp-new-test-binary.yaml

```diff
@@ -0,0 +1,934 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: NAMESPACE
+  required: true
+- name: IMAGE_FORMAT
+- name: IMAGE_INSTALLER
+  required: true
+- name: IMAGE_TESTS_KUBERNETES
```

Comment:
testing w/ tests-kubernetes for now,
next will test w/ extended-platform-tests and this PR: https://github.com/openshift/openshift-tests/pull/12
then will rename extended-platform-tests to openshift-tests (at that point this template will not be needed)